src/plugins/intel_gpu/tests/functional/concurrency/gpu_concurrency_tests.cpp

```diff
@@ -242,20 +240,15 @@ TEST(smoke_InferRequestDeviceMemoryAllocation, usmHostIsNotChanged) {
     // Modify tensor somehow and save as a reference values
     ov::test::utils::fill_tensor_random(output_tensor2);
 
-    std::vector<float> ref_values;
-    ref_values.resize(output_tensor2.get_byte_size());
-    std::memcpy(ref_values.data(), output_tensor2.data(), output_tensor2.get_byte_size());
+    ov::Tensor ref_tensor(output_tensor2.get_element_type(), output_tensor2.get_shape());
+    output_tensor2.copy_to(ref_tensor);
```

Comment:
Or use memcpy is simpler than `copy_to`

Reply:
This change is made in this PR. I removed these commits here